atlasdb-client/src/test/java/com/palantir/atlasdb/logging/KeyValueServiceLogArbitratorTest.java

```diff
@@ -43,4 +44,19 @@ public void allUnsafeArbitratorMarksRowNamesAsUnsafe() {
     public void allUnsafeArbitratorMarksColumnNamesAsUnsafe() {
         assertThat(ALL_UNSAFE.isColumnNameSafe(TABLE_REFERENCE, COLUMN_NAME)).isFalse();
     }
+
+    @Test
+    public void allSafeArbitratorMarksTableNamesAsUnsafe() {
+        assertThat(ALL_SAFE.isTableReferenceSafe(TABLE_REFERENCE)).isTrue();
+    }
+
+    @Test
+    public void allSafeArbitratorMarksRowNamesAsUnsafe() {
+        assertThat(ALL_SAFE.isRowComponentNameSafe(TABLE_REFERENCE, ROW_NAME)).isTrue();
+    }
+
+    @Test
+    public void allSafeArbitratorMarksColumnNamesAsUnsafe() {
+        assertThat(ALL_SAFE.isColumnNameSafe(TABLE_REFERENCE, COLUMN_NAME)).isTrue();
+    }
```

Comment:
I think these test names should end in `Safe`

Reply:
oops!